adobe-esign-connector/src_hd/com/axonivy/connector/adobe/esign/connector/AdminSetup/AdminSetup.xhtml

```diff
@@ -0,0 +1,162 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/frame-10-full-width.xhtml">
+		<ui:define name="title">AdminSetup</ui:define>
+		<ui:define name="content">
+			<f:event listener="#{logic.preRender}" type="preRenderComponent" />
+			<h2>
+				#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/AdminSetup/Title')}
+			</h2>
+
+			<h:form id="form">
+				<p:fieldset
+					legend="#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/AdminSetup/General')}">
+					<div class="ui-fluid ">
+						<div class="p-field p-grid p-align-baseline">
+							<div class="p-col-12 p-md-6">
+								<p:outputLabel for="host"
+									value="#{adminSetupBean.getVariableName('HOST')}" />
+							</div>
+							<div class="p-col-12 p-md-6">
+								<p:inputText id="host" value="#{data.host}" placeholder="" />
+							</div>
+						</div>
+						<div class="p-field p-grid p-align-baseline">
+							<div class="p-col-12 p-md-6">
+								<p:outputLabel for="returnPage"
+									value="#{adminSetupBean.getVariableName('RETURN_PAGE')}" />
+							</div>
+							<div class="p-col-12 p-md-6">
+								<p:inputText id="returnPage" value="#{data.returnPage}" />
+							</div>
+						</div>
+					</div>
+				</p:fieldset>
+
+				<p:fieldset
+					legend="#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/AdminSetup/IntegrationKey')}" >
+					<div class="ui-fluid">
+						<div class="p-field p-grid p-align-baseline">
+							<div class="p-col-12">
+								<p:staticMessage severity="info" detail="#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/AdminSetup/IntegrationKeyInfo')}" />
+							</div>
+						</div>
+						<div class="p-field p-grid p-align-baseline">
+							<div class="p-col-12 p-md-6 ">
+								<p:outputLabel for="integrationKey"
+									value="#{adminSetupBean.getVariableName('INTEGRATION_KEY')}" />
```

Comment:
Hi @stefanmasek
I think it would be better if we could put the output label value in cms rather than using the variable to store it.
You can check the Demo.xhtml in adobe-esign-connector-demo for references:
<p:outputLabel value="#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/demo/Demo/PDFFileToSign')}" />

Reply:
Hi @ntqdinh-axonivy, the outputlabel value is not stored in the Variable, the value is the Variable name. The Admin Setup page is for easier setup of the Variables and OAuth token so I think the output labels should have the exact Variable name 😄